webapp/src/components/ParcelCard/ParcelCard.css

```diff
@@ -36,6 +21,22 @@
   }
 }
 
+.ParcelCard .mortgage-status {
```

Comment:
Oh, Is there any reason for placing the media query before no-media-query CSS rules? I prefer to have media queries at the bottom of the file as we have in other files

Reply:
This is not the only place in the code we do this (see `Contribution.css` for example), there are pros an cons of having a desktop-first mobile-last css layout, on one hand is intuitive as you say, but on the other hand for instance when a file grows too big and there are classes affected by media queries the desktop and the mobile rules end up far apart making it prone to bugs due to updating one class and not the other. 

I don't know what's the best solution, we should probably come up with something at least and write it in out coding guidelines, but for the moment I don't think this PR is the place to do so, so I added an issue to continue this discussion: https://github.com/decentraland/marketplace/issues/443